experimental/ag-ui/front-end/src/components/LogsVisualization.css

```
@@ -47,385 +47,183 @@
-.log-row:nth-child(even):hover {
-  background-color: #f1f3f4;
-}
-
-/* Cell styles */
 .log-cell {
-  padding: 0.5rem;
-  border-right: 1px solid #f1f3f4;
+  padding: 0.4rem 0.5rem;
+  border-right: 1px solid var(--border-muted);
   vertical-align: top;
   word-wrap: break-word;
```

Stylelint (17.7.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)